Olha_Lesiuk/task_5.5.py

```diff
@@ -0,0 +1,4 @@
+import re
+text = ''.join(open("/Users/olya/Downloads/text_file/text").readlines())
+sentences = re.split(r' *[\.\?!][\'"\)\]]* *', text)
```

Comment:
The Hubble.Space.Telescope is not a sentence.

Reply:
The Hubble.Space.Telescope (often referred to as HST or Hubble) is a space telescope that was launched into low Earth orbit in 1990 and remains in operation....

This is one sentence

Reply:
decided to use if __name__ == '__main__' function

Reply:
It still split The Hubble.Space.Telescope  as separate sentences